package/yast2-firewall.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-firewall
-Version:        4.1.7
+Version:        4.1.8
 Release:        0
```

Comment:
It depends on this yast/yast-yast2#889, isn't it? In other case the attributes will be exported, so what about requiring also the version used in that PR when merged?.